samples/client/petstore/typescript-fetch/builds/allOf-nullable/models/Club.ts

```diff
@@ -31,24 +31,22 @@ export interface Club {
      * @type {Owner}
      * @memberof Club
      */
-    owner?: Owner | null;
+    owner?: Owner;
 }
 
 /**
  * Check if a given object implements the Club interface.
  */
 export function instanceOfClub(value: object): boolean {
-    let isInstance = true;
-
-    return isInstance;
+    return true;
 }
 
 export function ClubFromJSON(json: any): Club {
     return ClubFromJSONTyped(json, false);
 }
 
 export function ClubFromJSONTyped(json: any, ignoreDiscriminator: boolean): Club {
-    if ((json === undefined) || (json === null)) {
+    if (json === undefined || json === null) {
```

Comment:
should this be replaced with runtime.exists instead according to the PR description?

Reply:
`exists` accepts a dictionary and a key, it tests whether the key exists in the dictionary, so it won't be possible to use in this case.